<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherSettingsDialog</class>
 <widget class="QWidget" name="OtherSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbDontShowSplash">
        <property name="text">
         <string>DontShowSplash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbNativeFileDialog">
        <property name="text">
         <string>UseNativeFileDialog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbNativeTitile">
        <property name="text">
         <string>UseNativeTitleBar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAppData">
        <property name="text">
         <string>OpenAppData</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Log</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="cbLogLevel"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblLevel">
        <property name="text">
         <string>Level</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblCount">
        <property name="text">
         <string>Count</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Update</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cbCheckWhenStartup">
        <property name="text">
         <string>CheckWhenStartup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
